types/src/front/api_handlers/public/assistant.ts

```diff
@@ -1,12 +1,13 @@
 import * as t from "io-ts";
+import { NonEmptyString } from "io-ts-types";
 
 export const PublicPostMessagesRequestBodySchema = t.intersection([
   t.type({
     content: t.string,
     mentions: t.array(t.type({ configurationId: t.string })),
     context: t.type({
-      timezone: t.string,
-      username: t.string,
+      timezone: NonEmptyString,
```

Comment:
We also want to validate timezone manually to make sure it's a valid one on top of this non empty validation?